plugins/net_plugin/net_plugin.cpp

```diff
@@ -779,12 +788,12 @@ namespace eosio {
 
       fc::variant_object get_logger_variant()  {
          fc::mutable_variant_object mvo;
-         mvo( "_name", peer_name());
-         std::lock_guard<std::mutex> g_conn( conn_mtx );
-         mvo( "_id", conn_node_id )
-            ( "_sid", conn_node_id.str().substr( 0, 7 ) )
-            ( "_ip", remote_endpoint_ip )
-            ( "_port", remote_endpoint_port )
+         mvo( "_name", log_p2p_address)
```

Comment:
I don't see `log_p2p_address` is assigned somewhere

Reply:
Nice catch!

Reply:
Pushed fix.